src/UseCases/clients.js

```diff
@@ -0,0 +1,49 @@
+const Clients = require('../Models/clients');
+const bcrypt = require('../Lib/bcrypt');
+const jwt = require('../Lib/jwt');
+const sgMail = require('../Lib/sendgrid')
+
+function getClients(){
+   return Clients.find({})
+}
+
+function getById(idClient){
+   return Clients.findById(idClient)
+}
+
+ async function create(dataClients){
+   const {name, email, lastName} = dataClients
+   
+   const userFound = await getClients.findOne({email:email})
+   if (!userFound) throw new Error('Not permision to create, this client already exist') 
```

Comment:
@KristomeisterDev la condicional sería más bien preguntar si ya existe un usuario regresar el error. No si no se encuentra

```suggestion
   if (userFound) throw new Error('Not permision to create, this client already exist') 
```